subsys/profiler/Kconfig

```diff
@@ -9,10 +9,13 @@ menuconfig PROFILER
 
 if PROFILER
 
-config PROFILER_MAX_NUMBER_OF_CUSTOM_EVENTS
-	int "Maximum number of stored custom event types"
+config PROFILER_MAX_NUMBER_OF_EVENTS
+	int "Maximum number of stored event types"
 	default 32
-	range 0 32
+	range 0 255
+	help
+	  Maximum number of stored event types.
+	  Note that there are two hidden events for tracking event processing start and stop in Event Manager.
```

Comment:
The note should be. "You need one profiler event available for a single event manager event plus two additional events that are used for tracking event manager event execution (in case such tracking is enabled tracking)".